packages/ihe-gateway-sdk/src/__tests__/ihe-inbound-lambdas.test.ts

```diff
@@ -0,0 +1,23 @@
+import { inboundPatientDiscoveryReqSchema } from "../models/patient-discovery/patient-discovery-requests";
+import { inboundDocumentQueryReqSchema } from "../models/document-query/document-query-requests";
+import { inboundDocumentRetrievalReqSchema } from "../models/document-retrieval/document-retrieval-requests";
+import { testInboundXCPD, testInboundDQ, testInboundDR } from "./constants";
+
+describe("Inbound Patient Discovery Request Validation", () => {
+  it("should successfully validate the testInboundXCPD object with the inboundPatientDiscoveryReqSchema", () => {
+    const result = inboundPatientDiscoveryReqSchema.safeParse(testInboundXCPD);
+    expect(result.success).toBe(true);
+  });
```

Comment:
We also need to test w/ a payload missing `cxId`, based on the changes on `baseRequestSchema`, right?

I'd have a test w/ a regular payload, one w/o the optional properties.